File: DataSummitSolution/DataSummitUI/src/app/projects/models/rectanglepairs.ts
import { RectanglePair } from "./rectanglepair"

export class RectanglePairs {

    ProjectId: number
    Height: number
    Width: number
    Pairs: RectanglePair[]
    // Pairs: string | RectanglePair[]

    constructor(projectId?: number, height?: number, 
        width?: number, rectanglePairs?: RectanglePair[]) {
            this.ProjectId = projectId || 0
            this.Height = height || 0
            this.Width = width || 0 
            //this.Pairs = rectanglePairs || ''
            this.Pairs = rectanglePairs
        }
}

File: DataSummitSolution/AzureFunctions/Models/Recognition/RectanglePairs.cs
using System.Collections.Generic;

namespace AzureFunctions.Models.Recognition
{
    public class RectanglePairs
    {
        public int CompanyId { get; set; }
        public long Height { get; set; }
        public long Width { get; set; }
        public string TemplateName { get; set; }
        public List<RectanglePair> Pairs { get; set; }

        public RectanglePairs()
        {
            Pairs = new List<RectanglePair>();
        }
    }
}
